front/app/services/initiativeStatusChanges.ts

```diff
@@ -51,14 +55,14 @@ export async function updateInitiativeStatusWithExistingFeedback(
       },
     }
   );
-  streams.fetchAllWith({
-    apiEndpoint: [
-      `${API_PATH}/initiatives`,
-      `${API_PATH}/initiatives/${initiativeId}`,
-      `${API_PATH}/initiatives/filter_counts`,
-      `${API_PATH}/initiatives/${initiativeId}/allowed_transitions`,
-    ],
-  });
+
+  queryClient.invalidateQueries(initiativesKeys.lists());
```

Comment:
Nice, we might have to do something similar if there are any other places where we need to invalidate after full testing in our branch